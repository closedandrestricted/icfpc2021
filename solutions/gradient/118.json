{"vertices":[[182,27],[25,100],[210,19],[30,81],[36,137],[38,134],[25,62],[59,121],[51,94],[42,163],[102,26],[154,35],[90,79],[105,65],[49,94],[114,1],[105,50],[189,36],[140,55],[62,144],[58,117],[16,160],[24,86],[127,54],[115,67],[103,86],[73,99],[60,184],[169,59],[120,78],[122,46],[123,21],[111,98],[37,135],[10,137],[121,77],[83,80],[32,190],[144,68],[95,105],[84,126],[140,64],[102,72],[143,63],[122,100],[42,207],[59,134],[114,123],[90,153],[144,39],[52,178],[134,90],[77,170],[141,89],[59,183],[110,123],[88,149],[167,54],[94,171],[164,82],[75,165],[73,167],[71,191],[112,147],[113,99],[160,104],[113,182],[111,174],[139,222],[136,47],[171,145],[91,184],[64,212],[144,122],[140,65],[154,253],[110,173],[100,166],[90,205],[224,201],[161,49],[134,191],[132,39],[136,231],[197,178],[91,181],[138,161],[102,193],[251,220],[174,149],[158,210],[228,104],[226,194],[167,179],[259,78],[202,129],[180,208],[195,164],[206,162],[231,65],[209,92],[208,193],[207,226],[223,148],[233,118],[234,114],[261,104],[262,139]]}