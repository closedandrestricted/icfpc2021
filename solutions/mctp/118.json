{"vertices":[[3,23],[21,55],[1,51],[26,36],[2,84],[2,32],[7,38],[23,44],[5,33],[27,75],[22,31],[28,38],[7,51],[28,50],[3,38],[4,51],[4,51],[2,25],[7,25],[5,58],[27,42],[2,84],[8,63],[13,31],[5,31],[4,37],[1,62],[3,77],[5,55],[9,56],[13,5],[3,29],[3,36],[2,33],[1,60],[9,54],[12,56],[27,92],[2,81],[4,84],[23,70],[28,42],[4,74],[1,29],[27,69],[22,56],[1,55],[5,61],[19,72],[14,53],[5,84],[7,58],[22,64],[7,77],[3,78],[24,94],[24,93],[27,30],[5,84],[14,55],[2,64],[23,70],[1,61],[2,84],[3,105],[30,70],[28,85],[2,57],[39,66],[27,61],[4,81],[6,88],[23,54],[1,45],[2,81],[5,66],[3,110],[15,69],[25,80],[52,132],[2,55],[13,82],[27,71],[17,41],[32,103],[9,61],[32,104],[3,46],[25,116],[33,66],[37,63],[1,47],[2,89],[8,80],[2,86],[5,83],[2,49],[2,49],[1,51],[31,93],[32,58],[3,17],[31,32],[2,83],[2,79],[1,47],[1,48],[2,83]]}