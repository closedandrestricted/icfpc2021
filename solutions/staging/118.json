{"vertices":[[89,82],[18,119],[94,110],[22,100],[37,180],[73,143],[17,81],[49,140],[44,113],[29,154],[131,25],[117,85],[100,99],[98,78],[41,113],[108,11],[101,59],[113,91],[133,68],[56,162],[47,137],[6,165],[40,87],[121,68],[82,55],[95,85],[65,118],[15,193],[142,101],[113,91],[124,52],[124,27],[115,113],[31,154],[7,141],[115,90],[86,66],[34,213],[119,88],[88,119],[75,139],[97,76],[96,84],[136,76],[114,113],[33,216],[54,157],[104,91],[83,191],[142,48],[46,193],[135,105],[50,218],[134,104],[57,199],[103,136],[81,162],[160,68],[95,176],[156,95],[75,174],[66,180],[66,204],[105,160],[107,112],[139,80],[73,178],[105,187],[117,228],[183,88],[164,159],[84,197],[57,225],[200,153],[184,87],[150,223],[103,186],[63,196],[82,218],[223,196],[203,105],[130,200],[174,112],[156,251],[200,169],[93,196],[128,171],[85,182],[248,217],[165,173],[153,221],[214,144],[218,198],[156,188],[254,139],[178,141],[176,213],[185,175],[199,167],[227,124],[195,109],[204,201],[203,234],[217,169],[215,79],[218,133],[243,66],[227,97]]}