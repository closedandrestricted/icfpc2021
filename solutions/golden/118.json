{"vertices":[[175,40],[19,117],[203,32],[23,98],[35,147],[74,139],[18,79],[81,162],[45,109],[35,175],[133,23],[147,48],[109,96],[98,78],[42,110],[108,11],[99,65],[185,52],[134,68],[58,159],[48,133],[9,173],[42,84],[121,68],[81,56],[95,99],[65,119],[54,198],[162,72],[113,91],[134,50],[124,27],[116,112],[32,155],[6,149],[115,90],[97,119],[26,199],[137,67],[88,119],[75,139],[96,77],[115,109],[136,76],[114,113],[34,218],[55,156],[118,137],[83,161],[145,45],[47,191],[160,59],[72,184],[134,104],[53,196],[104,136],[81,162],[160,68],[91,178],[156,95],[71,176],[67,180],[65,204],[105,160],[128,132],[139,80],[106,195],[105,187],[151,223],[189,78],[95,186],[84,197],[58,225],[128,170],[184,85],[156,256],[103,186],[96,213],[83,218],[225,201],[208,96],[128,203],[203,66],[129,246],[202,174],[88,194],[131,173],[103,188],[251,221],[167,162],[150,224],[221,117],[220,202],[159,193],[248,87],[195,142],[177,219],[186,176],[220,164],[219,93],[188,106],[203,203],[205,236],[215,160],[213,131],[215,124],[224,161],[250,138]]}